client/packages/common/src/utils/mappers/gender.ts

```diff
@@ -1,6 +1,15 @@
 import { LocaleKey, GenderTypeNode } from '@openmsupply-client/common';
 import { noOtherVariants } from '../types';
 
+export const isSupportedGender = (gender: GenderTypeNode) => {
+  return ![
+    GenderTypeNode.TransgenderFemaleHormone,
+    GenderTypeNode.TransgenderFemaleSurgical,
+    GenderTypeNode.TransgenderMaleHormone,
+    GenderTypeNode.TransgenderMaleSurgical,
+  ].includes(gender);
+};
+
```

Comment:
If we're specifically trying to resolve the error in the issue. Then filtering out the options is the easiest and quickest solution. 

If we want to fully remove these from the enum. I think would require further discussions, a few migrations, and type regeneration. Mainly the discussion bit - if its still going to be used or is currently used